### ISSUE TITLE
EmptyBlockCheck crashes for some annotation defaults

### ISSUE THREAD

**Issue by denizarsan** [checkstyle/checkstyle#4631]
$ javac C.java

$ cat C.java
```java
@interface C {
    String[] m() default "";
}
```
$ cat config.xml
```
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.3//EN" "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
<module name="Checker">
  <module name="TreeWalker">
    <module name="EmptyBlock">
      <property name="tokens" value="LITERAL_DEFAULT"/>
    </module>
  </module>
</module>
```
$ java -jar checkstyle-8.0-all.jar -c config.xml C.java
```
Starting audit...
com.puppycrawl.tools.checkstyle.api.CheckstyleException: Exception was thrown while processing C.java
        at com.puppycrawl.tools.checkstyle.Checker.processFiles(Checker.java:295)
        at com.puppycrawl.tools.checkstyle.Checker.process(Checker.java:213)
        at com.puppycrawl.tools.checkstyle.Main.runCheckstyle(Main.java:430)
        at com.puppycrawl.tools.checkstyle.Main.runCli(Main.java:362)
        at com.puppycrawl.tools.checkstyle.Main.main(Main.java:177)
Caused by: java.lang.NullPointerException
        at com.puppycrawl.tools.checkstyle.checks.blocks.EmptyBlockCheck.findLeftCurly(EmptyBlockCheck.java:246)
        at com.puppycrawl.tools.checkstyle.checks.blocks.EmptyBlockCheck.visitToken(EmptyBlockCheck.java:142)
        at com.puppycrawl.tools.checkstyle.TreeWalker.notifyVisit(TreeWalker.java:368)
        at com.puppycrawl.tools.checkstyle.TreeWalker.processIter(TreeWalker.java:505)
        at com.puppycrawl.tools.checkstyle.TreeWalker.walk(TreeWalker.java:310)
        at com.puppycrawl.tools.checkstyle.TreeWalker.processFiltered(TreeWalker.java:182)
        at com.puppycrawl.tools.checkstyle.api.AbstractFileSetCheck.process(AbstractFileSetCheck.java:78)
        at com.puppycrawl.tools.checkstyle.Checker.processFile(Checker.java:316)
        at com.puppycrawl.tools.checkstyle.Checker.processFiles(Checker.java:286)
        ... 4 more
Checkstyle ends with 1 errors.
```
---------------
While the report is for the most recent Checkstyle version, this also fails on checkstyle-7.8.2-all.jar but passes on checkstyle-7.6.1-all.jar, so there was some regression between 7.6.1 and 7.8.2.

**Comment by rnveach** [comment]
NPE was created in 7.7 release.
Most likely caused by Issue #4159.

**Comment by romani** [comment]
@denizarsan , if you run checkstyle on your opensource project , please let us know we could add your project to regression testing process.

**Comment by rnveach** [comment]
This appears to be a duplicate of #4472 and was already fixed for 8.1 .
I am closing the issue.
